docs/src/pages/[platform]/getting-started/migration/migration.react.mdx

```diff
@@ -79,6 +79,53 @@ Hopefully this change won't affect your code but will allow for more customizati
   The TextAreaField component does apply 1 style prop directly on the `textarea` element: `resize`. We felt this one makes sense to apply direclty on the the `textarea` element and not the wrapper element.
 </Alert>
 
+#### Automatic signin on signup logic moved to `aws-amplify`. 
+
+##### 1. Update the handleSignUp function override call
+
+If you are overriding `Auth.signUp`, update the override function call to include the `autoSignIn` option set to enabled. If this change is not made, users will not be automatically signed in on signup.
+
+```diff
+ async handleSignUp(formData) {
+  let { username, password, attributes } = formData;
+  // custom username
+  username = username.toLowerCase();
+  attributes.email = attributes.email.toLowerCase();
+  return Auth.signUp({
+    username,
+    password,
+    attributes,
++   autoSignIn: {
++     enabled: true
++   }
+  });
+}
+
+```
+
+##### 2. Update to the latest version of the `aws-amplify` library.
+
+Update to the latest version of the `aws-amplify` library. You will need at least version `^5`. 
+
+<Tabs>
+<TabItem title="npm">
+
+```shell
+npm install aws-amplify@latest
+```
+
+</TabItem>
+<TabItem title="yarn">
+```shell
+yarn add aws-amplify@latest
+```
+
+</TabItem>
+</Tabs>
```

Comment:
Should we just move this to the top of the 3-4 migration section since ui@4 needs libraries@5? Maybe we can then remove the level 5 headings from this section.

Reply:
Fixed